.github/ISSUE_TEMPLATE/improvement.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Improvement Suggestion
```

Comment:
This looks the same as the "feature" template. While the difference between improvement and feature is subtle, it's not worth making a different template for it, IMO. 
I don't feel strongly about it, though. Feel free to keep it if you do.

Reply:
The way I see it, a "feature" is something new while an "improvement" is, well, an improvement to something that already exists.

Reply:
I always thoght of "Feature Requests" as suggestions of new functionality while Improvements are suggestions about, well, improving existing features...

Reply:
But, yes, they do represent related concepts in the end.

Reply:
We can try with both. It'll be easy to remove if a wrong one gets picked too often.

Reply:
I agree.